src/Helper.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Sideways\Classic;
+
+class Helper
+{
+    function getSiteSchemaMarkup(): ?string
+    {
+        $seoService = new Seo\SeoService();
```

Comment:
isn't better to use `use Sideways\Classic\Seo\SeoService;` at the top instead of this, besides it's duplicated `Seo\` + we are using on the rest of file or other projects `use Class\...` for consistence.